Makefile

```diff
@@ -20,8 +25,12 @@ dependencies:
 	@glide --quiet update
 
 test:
-	@echo ">> running tests"
+	@echo ">> starting local redis-server on port $(REDIS_PORT)"
```

Comment:
I'd not make this part of the tests.
E.g. I run my local dev redis instance in a docker container and don't need that step. I think it's fair to assume that whoever runs the test has a local redis instance running.